lib/CredentialsWrapper.jsx

```diff
@@ -0,0 +1,53 @@
+import localForage from 'localforage';
+
+const PARTNER_DETAILS = {
+    CREDENTIALS_KEY: 'DEVICE_SESSION_CREDENTIALS',
+    EXPENSIFY_PARTNER_PREFIX: 'expensify.',
+    PARTNER_NAME: 'chat-expensify-com',
```

Comment:
NAB: but this is is already a const in `lib/CONST.jsx`

Reply:
Updated it to be `LOGIN_PARTNER_DETAILS`.

Reply:
<img width="861" alt="Screenshot 2023-01-05 at 2 53 47 PM" src="https://user-images.githubusercontent.com/35863227/210745878-e3b5682c-16e5-40f1-898e-2181eff21b31.png">

This is how it looks currently under the device list and I suggest we change the device type and name to that describes where these credentials were generated so it will be easy to revoke access.

I can create a followup issue to look into it.

Reply:
There's maybe a few libraries that can allow us to describe is as `iPhone 13 Pro` we should maybe look into as a nice follow-up.

Reply:
Created an issue here for using nice naming: https://github.com/Expensify/Expensify/issues/254636 that I will explore later on.